Move event calendar and update community page (#387)
trinodb/trino.io: community.md

        </div>
      </div>
    </div>
    <div class="community-content-section">

>       <h2>Event Calendar</h2>

Reviewer: ```suggestion
      <h2>Calendar</h2>
```

>       Events for Trino Meetups, Trino Community Broadcasts, and other Trino

Reviewer: ```suggestion
      Community events such as Trino Meetups, Trino Community Broadcasts, and other Trino
```

>       events you don't want to miss. To follow along, add this calendar by

Reviewer: ```suggestion
      gatherings you don't want to miss. To follow along, add this calendar by
```
